tests/zfs-tests/tests/perf/regression/random_reads.ksh

```diff
@@ -85,9 +85,16 @@ log_must fio $FIO_SCRIPTS/mkfiles.fio
 lun_list=$(pool_to_lun_list $PERFPOOL)
 log_note "Collecting backend IO stats with lun list $lun_list"
 if is_linux; then
-        export collect_scripts=("zpool iostat -lpvyL $PERFPOOL 1" "zpool.iostat"
-            "vmstat 1" "vmstat" "mpstat -P ALL 1" "mpstat" "iostat -dxyz 1"
-            "iostat")
+	export collect_scripts=("zpool iostat -lpvyL $PERFPOOL 1" "zpool.iostat"
+	    "vmstat 1" "vmstat" "mpstat -P ALL 1" "mpstat" "iostat -dxyz 1"
+	    "iostat")
+
+	if [[ -x /usr/bin/perf ]]; then
```

Comment:
We don't fail the test if a collection script fails, because it's still worth finishing the test if (for example) a DTrace script errors out due to systemic unresponsiveness. So now that we're not checking for an environment variable, there's no need to treat this clause separately.

Reply:
Good point. I'll remove if statement and include perf invocation in collect_scripts